@xen-orchestra/lite/src/views/settings/SettingsView.vue

```diff
@@ -178,10 +178,10 @@ h5 {
     flex-direction: column;
     gap: 1.6em;
     &.selected {
-      color: var(--color-normal-txt-base);
+      color: var(--color-info-txt-base);
 
       img {
-        outline: solid 2px var(--color-normal-txt-base);
+        outline: solid 2px var(--color-info-txt-base);
```

Comment:
shouldn't it be in rem?
```suggestion
        outline: solid 0.2rem var(--color-info-txt-base);
```

Reply:
Yes, indeed, but there are also other values in this component and other XO Lite components that need to be updated, and I'm not sure it has to be done in this PR.

Reply:
Yes I wasn't sure it should be done here either
I think we can do another PR to update components later